AWapi.playground/Contents.swift

```diff
@@ -0,0 +1,101 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+import PlaygroundSupport
+
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+var str = "Hello, playground"
+
+import CoreData
+import Foundation
+
+let baseURL = "https://www.americanwhitewater.org/content/"
+
+let riverURL = baseURL + "River/search/.json"
+
+struct AWReachInfo: Codable {
+    //swiftlint:disable:next identifier_name
+    let id: Int //
```

Comment:
Why do these all have empty comments after them?

Reply:
Remnants of debugging which property JSONDecoder.decode was having a hard time with.